cupy/cuda/memory.pyx

```diff
@@ -1514,9 +1514,6 @@ cdef class MemoryPool:
         return mp.get_limit()
 
 
-cdef bint MemoryAsyncHasStat = (runtime.driverGetVersion() >= 11030)
```

Comment:
For the record this was my bad. I thought it's safe to query the version without initializing the CUDA context, but apparently I didn't verify this assumption...🤦🏻‍♂️

Reply:
no worries! :)